### PR TITLE
OCM-7805 | feat: Add delete command

### PR DESCRIPTION
**Pull request by hunterkepley**
https://issues.redhat.com/browse/OCM-7805

Adds delete command to rosa-support CLI for QE/UI team usage

### CODE REVIEW

File: cmd/rosa-support/create/cmd.go [hunk 1]
```diff
@@ -1,3 +1,19 @@
+/*
+Copyright (c) 2019 Red Hat, Inc.
```
Comment (davidleerh) [inline cmd/rosa-support/create/cmd.go]: 2024

File: cmd/rosa-support/delete/cmd.go [hunk 2]
```diff
@@ -0,0 +1,35 @@
+/*
+Copyright (c) 2019 Red Hat, Inc.
```
Comment (davidleerh) [inline cmd/rosa-support/delete/cmd.go]: 2024

File: cmd/rosa-support/delete/tag/cmd.go [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+	awsClient "github.com/openshift-online/ocm-common/pkg/aws/aws_client"
+	logger "github.com/openshift-online/ocm-common/pkg/log"
+	"github.com/spf13/cobra"
+)
+
+var args struct {
+	region      string
+	resourceID  string
+	tagKey      string
+	tagValue    string
```
Comment (davidleerh) [inline cmd/rosa-support/delete/tag/cmd.go:15]: i dont think the tag value should be something that is required during delete :)

File: cmd/rosa-support/delete/tag/cmd.go [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+	profileName string
+}
+var Cmd = &cobra.Command{
+	Use:   "tag",
+	Short: "Delete tag",
+	Long:  "Delete tag.",
+	Example: `  # Delete a tag from the resource
+  rosa-support delete tag --resource-id <vpc id> --region us-east-2 --tag-key key --tag-value value`,
```
Comment (davidleerh) [inline cmd/rosa-support/delete/tag/cmd.go]: tag value shouldn't be an option / arg to require during delete tag :)

File: cmd/rosa-support/delete/tag/cmd.go (hunk 3 above)
Comment (davidleerh) [reply cmd/rosa-support/delete/tag/cmd.go:15]: Due to possible issues of missed deletion:
https://docs.aws.amazon.com/cli/latest/reference/ec2/delete-tags.html#:~:text=If%20you%20specify%20the%20empty%20string%20as%20the%20tag%20value%2C%20the%20tag%20is%20deleted%20only%20if%20the%20tag%27s%20value%20is%20the%20empty%20string.%20The%20following%20delete%2Dtags%20example%20specifies%20the%20empty%20string%20as%20the%20tag%20value%20for%20the%20tag%20to%20delete.

**Review by ciaranRoche — APPROVED** [review]

**Pull request merged**